src/cascadia/TerminalApp/DefaultProfileUtils.cpp

```diff
@@ -15,19 +15,21 @@ static constexpr std::wstring_view PACKAGED_PROFILE_ICON_EXTENSION{ L".png" };
 // - name: the name of the new profile.
 // Return Value:
 // - A Profile, ready to be filled in
-TerminalApp::Profile CreateDefaultProfile(const std::wstring_view name)
+winrt::TerminalApp::Profile CreateDefaultProfile(const std::wstring_view name)
 {
     const auto profileGuid{ Microsoft::Console::Utils::CreateV5Uuid(TERMINAL_PROFILE_NAMESPACE_GUID,
                                                                     gsl::as_bytes(gsl::make_span(name))) };
-    TerminalApp::Profile newProfile{ profileGuid };
+    //auto newProfile = winrt::make<winrt::TerminalApp::Profile>(profileGuid);
+    winrt::TerminalApp::Profile newProfile;
+    newProfile.Guid({ profileGuid });

```

Comment:
I try to do `winrt::make` here and I get a bunch of errors in base.h. This feels wrong...